.github/workflows/reusable-releasability.yaml

```diff
@@ -76,9 +76,8 @@ jobs:
           # The following pushes the stdout of buoy into $CHECK_MESSAGE
           CHECK=0
           echo 'CHECK_MESSAGE<<EOF' >> "$GITHUB_ENV"
-          buoy check go.mod --release "${RELEASE}" --module-release "${MODULE_RELEASE}"" --domain knative.dev --verbose >> "$GITHUB_ENV" 2>&1 || CHECK=$?
+          buoy check go.mod --release "${RELEASE}" --module-release "${MODULE_RELEASE}" --domain knative.dev --verbose >> "$GITHUB_ENV" 2>&1 || CHECK=$?
           echo 'EOF' >> "$GITHUB_ENV"
-
```

Comment:
why we removing this line?

Reply:
Oh didn't notice my linter did that, let me revert